scripts/assign-colors

```diff
@@ -23,6 +23,16 @@ if __name__ == '__main__':
         metadata. If the metadata includes a 'focal' column that only contains
         boolean values, then restrict colors to traits for rows where 'focal'
         is set to True.""")
+    parser.add_argument('--ignore-categories', type=str, default=[], nargs='*',
+        help="""Do not create colors for these categories even if they are
+        included in the metadata and ordering TSV. This is useful for ignoring
+        categories in specific builds even if they share the same default
+        ordering TSV.""")
+    parser.add_argument('--force-include-categories', type=str, default=[],nargs='*',
+        help="""Force include all color orderings for these categories even if
+        there are traits not included in the metadata TSV. This is useful for
+        creating colorings for traits not (yet) present in metadata to solve
+        bootstrapping issue.""")
```

Comment:
I have no idea what a "bootstrapping issue" is. I suspect this is a valid use-case, but perhaps adjust the argument help text?